helpdesk_mgmt_ticket_stage_auto/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# © 2021 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See https://www.gnu.org/licenses/lgpl-3.0.html
+{
+    "name": "Helpdesk Ticket Stage Auto",
+    "summary": """
+        Adds a programmed action to change the auto stage
+    """,
+    "version": "12.0.1.0.0",
+    "license": "LGPL-3",
+    "category": "After-Sales",
+    "author": "Solvos",
+    "website": "https://github.com/solvosci/slv-helpdesk",
+    "depends": [
+        "helpdesk_mgmt",
+        "helpdesk_mgmt_timesheet",
```

Comment:
Unnecessary dependency (it's already included):
```suggestion
        "helpdesk_mgmt_timesheet",
```